doc/api/deprecations.md

```diff
@@ -2542,6 +2542,24 @@ accordingly instead to avoid the ambigiuty.
 To maintain existing behaviour `response.finished` should be replaced with
 `response.writableEnded`.
 
+<a id="DEP0139"></a>
+### DEP0139: `process.umask()` with no arguments
+<!-- YAML
+changes:
+  - version:
+    - v14.0.0
+    - REPLACEME
```

Comment:
This should probably go before the v14.0.0.

Reply:
```suggestion
```